components/motor/gpio/motor_encoder_test.go

```diff
@@ -99,14 +98,23 @@ func TestMotorEncoder1(t *testing.T) {
 			test.That(tb, fakeMotor.PowerPct(), test.ShouldEqual, float32(1))
 		})
 
-		test.That(t, interrupt.Ticks(context.Background(), 99, nowNanosTest()), test.ShouldBeNil)
-		test.That(t, fakeMotor.Direction(), test.ShouldEqual, 1)
-
-		_motor.SetPower(context.Background(), .25, nil)
-		test.That(t, interrupt.Ticks(context.Background(), 1000, nowNanosTest()), test.ShouldBeNil) // go far!
+		errChan := make(chan error)
+		go func() {
+			ticksErr := interrupt.Ticks(context.Background(), 99, nowNanosTest())
+			errChan <- ticksErr
+		}()
+		time.Sleep(2 * time.Millisecond)
+		_motor.SetPower(context.Background(), -0.25, nil)
```

Comment:
Is the expectation that SetPower is processed before interrupt.Ticks() is done propagating 99 ticks to the encoder? If so, this is a race that will be very flaky. also, if i understanding this correctly, the go-routine will continue running after GoFor is killed and the position will continue incrementing...so the test for Position for <100 seems unnecessary...